<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rb_reg_cut">
       <property name="text">
        <string>Regular Hair Cut $15</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_senior_cut">
       <property name="text">
        <string>Senior Hair Cut  $13</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_shave">
       <property name="text">
        <string>Shave  $4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_trim">
       <property name="text">
        <string>Mustache Trim  $2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_beard">
       <property name="text">
        <string>Beard Trim  $4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_cost">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>Total Cost</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
